rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/RMRest.java

```diff
@@ -270,11 +271,27 @@ public RMState getState(@HeaderParam("sessionid") String sessionId) throws NotCo
     @GZIP
     @Path("monitoring")
     @Produces("application/json")
-    public RMStateDelta getInitialState(@HeaderParam("sessionid") String sessionId,
+    public RMStateDelta getRMStateDelta(@HeaderParam("sessionid") String sessionId,
             @HeaderParam("clientCounter") @DefaultValue("-1") String clientCounter) throws NotConnectedException {
         checkAccess(sessionId);
         long counter = Integer.valueOf(clientCounter);
-        return RMStateCaching.getRMInitialState(counter);
+        return RMStateCaching.getRMStateDetla(counter);
+    }
+
+    /**
+     * Returns the state of the RM, which does not include REMOVED node/nodesources
```

Comment:
Returns the full state of the RM, ...

could you add the javadoc in RMRestInterface instead ? (and also move other javadoc methods)

Reply:
Ok